extensions/indexes/lucene/src/org/exist/indexing/lucene/LuceneIndexWorker.java

```diff
@@ -487,6 +488,13 @@ protected void setOptions(Properties options, CommonQueryParserConfiguration par
             else
                 parser.setMultiTermRewriteMethod(MultiTermQuery.CONSTANT_SCORE_BOOLEAN_QUERY_REWRITE);
         }
+        option = options.getProperty(OPTION_LOWERCASE_EXPANDED_TERMS);
+        if (option != null) {
+            if (option.equalsIgnoreCase("yes"))
```

Comment:
still, swapping the string literal and the value, makes the code NullPointerException safe, so the != check can be left out.

```java
parser.setLowercaseExpandedTerms("yes".equalsIgnoreCase(option));
```